src/main/lrsql/admin/interceptors/ui.clj

```diff
@@ -1,7 +1,20 @@
 (ns lrsql.admin.interceptors.ui
-  (:require [ring.util.response :as resp]))
+  (:require [ring.util.response :as resp]
+            [io.pedestal.interceptor :refer [interceptor]]))
 
 (defn admin-ui-redirect
   "Handler function to redirect to the admin ui"
   [_]
   (resp/redirect "/admin/index.html"))
+
+(def get-env
+  "Provide select config data to client upon request"
+  (interceptor
+   {:name ::get-env
+    :enter
+    (fn get-env [ctx]
+      (let [{url-prefix :com.yetanalytics.lrs.pedestal.interceptor/path-prefix
+             enable-stmt-html :com.yetanalytics.lrs.pedestal.interceptor/statement-html?} ctx]
```

Comment:
Please shorten this with qualified namespaces!

Reply:
looks like the ns already is required, so it's an easy fix

Reply:
different ns, but yeah this is done